cesium_app/tests/test_util.py

```diff
@@ -80,3 +84,37 @@ def test_check_model_param_types():
     params = {"invalid_param_name": "some_value"}
     pytest.raises(ValueError, sklearn_models.check_model_param_types,
                   model_type, params)
+
+
+def test_prediction_to_csv_class():
+    """Test util.prediction_to_csv"""
+    with create_test_project() as p, create_test_dataset(p) as ds,\
+         create_test_featureset(p) as fs,\
+         create_test_model(fs, model_type='LinearSGDClassifier') as m,\
+         create_test_prediction(ds, m) as pred:
+        pred = xr.open_dataset(pred.file.uri)
+        assert util.prediction_to_csv(pred) ==\
+            [['ts_name', 'true_target', 'prediction\n'],
+             ['0', 'Mira', 'Mira\n'],
+             ['1', 'Classical_Cepheid', 'Classical_Cepheid\n'],
+             ['2', 'Mira', 'Mira\n'],
+             ['3', 'Classical_Cepheid', 'Classical_Cepheid\n'],
+             ['4', 'Mira', 'Mira\n']]
+
+
+def test_prediction_to_csv_regr():
+    """Test util.prediction_to_csv"""
+    with create_test_project() as p, create_test_dataset(p, label_type='regr') as ds,\
+         create_test_featureset(p, label_type='regr') as fs,\
+         create_test_model(fs, model_type='LinearRegressor') as m,\
+         create_test_prediction(ds, m) as pred:
+        pred = xr.open_dataset(pred.file.uri)
```

Comment:
This code gets quite dense, so some spaces would be helpful to segment the creation from the rest of the function (newline after "as pred").